.ci/jenkins_job_build.sh

```diff
@@ -88,20 +97,19 @@ then
 else
 	# Othewise we test an specific branch
 	# GIT_BRANCH env variable is set by the jenkins Github Plugin.
-	remote="${GIT_BRANCH/\/*/}"
-	branch="${GIT_BRANCH/*\//}"
+	export branch="${GIT_BRANCH/*\//}"
```

Comment:
do we need to check if branch is not empty ? 
Why we are exporting this variable will be used by other script?

Reply:
yes, this will be used by the `.ci/lib.sh` script